stonesoup/models/measurement/nonlinear.py

```diff
@@ -1248,3 +1250,294 @@ def pdf(self, state1, state2, **kwargs):
     def logpdf(self, *args, **kwargs):
         # As pdf replaced, need to go to first non GaussianModel parent
         return super(ReversibleModel, self).logpdf(*args, **kwargs)
+
+
+class IsotropicPlume(GaussianModel, MeasurementModel):
```

Comment:
I think this should be in a separate module, as it's very specific. `stonesoup.models.measurement.gas` possibly, which matches name used for *sensor*?

Reply:
This has been done in the latest commit.